src/controllers/softtakeover.cpp

```diff
@@ -60,14 +66,21 @@ bool SoftTakeoverCtrl::ignore(ControlObject* control, double newParameter) {
     return ignore;
 }
 
+void SoftTakeoverCtrl::ignoreNext(ControlObject* control) {
```

Comment:
style nit:  I've come to prefer the "test for error and return" method.  It reduces nesting and makes the flow more obvious:

if (control == NULL) {
 return;
}
SoftTakeover* pSt = m_softTakeoverHash.value(control);
if (pSt == NULL) {
  return;
}

pSt->ignoreNext();